libshaderc/include/shaderc/shaderc.h

```diff
@@ -255,6 +342,10 @@ void shaderc_compile_options_set_target_env(shaderc_compile_options_t options,
 void shaderc_compile_options_set_warnings_as_errors(
     shaderc_compile_options_t options);
 
+// Sets a resource limit.
+void shaderc_compile_options_set_limit(
+    shaderc_compile_options_t options, shaderc_limit limit, int value);
```

Comment:
I'm not particular clear about this part tho: should we use an explicitly sized integer type here?